Failing Installation on Windows Machine

Please answer the following before submitting your issue:

Note: Please include any substantial examples (debug session output,
stacktraces, etc) as linked gists.

1. What version of Delve are you using (`dlv version`)? Latest in github
2. What version of Go are you using? (`go version`)? V1.15
3. What operating system and processor architecture are you using? Windows 64
4. What did you do? Tried installing delve using: go get -u -v github.com/go-delve/delve/cmd/dlv
5. What did you expect to see? Successfull installation
6. What did you see instead?

```
github.com/go-delve/delve (download)
created GOPATH=C:\Users\kbuilder\Go; see 'go help gopath'
github.com/go-delve/delve/pkg/proc/macutil
github.com/go-delve/delve/vendor/golang.org/x/sys/internal/unsafeheader
github.com/go-delve/delve/vendor/github.com/shurcooL/sanitized_anchor_name
github.com/go-delve/delve/vendor/github.com/hashicorp/golang-lru/simplelru
github.com/go-delve/delve/vendor/github.com/konsorten/go-windows-terminal-sequences
github.com/go-delve/delve/vendor/github.com/mattn/go-isatty
github.com/go-delve/delve/vendor/gopkg.in/yaml.v2
github.com/go-delve/delve/pkg/goversion
github.com/go-delve/delve/vendor/github.com/cosiner/argv
github.com/go-delve/delve/pkg/dwarf/util
github.com/go-delve/delve/vendor/github.com/sirupsen/logrus
github.com/go-delve/delve/pkg/dwarf/regnum
github.com/go-delve/delve/pkg/elfwriter
github.com/go-delve/delve/pkg/astutil
github.com/go-delve/delve/pkg/version
github.com/go-delve/delve/vendor/golang.org/x/arch/arm64/arm64asm
github.com/go-delve/delve/pkg/terminal/colorize
github.com/go-delve/delve/vendor/golang.org/x/arch/x86/x86asm
github.com/go-delve/delve/pkg/dwarf/frame
github.com/go-delve/delve/pkg/dwarf/op
github.com/go-delve/delve/pkg/dwarf/line
github.com/go-delve/delve/pkg/logflags
github.com/go-delve/delve/vendor/golang.org/x/sys/windows
github.com/go-delve/delve/pkg/dwarf/godwarf
github.com/go-delve/delve/vendor/github.com/peterh/liner
github.com/go-delve/delve/vendor/go.starlark.net/internal/spell
github.com/go-delve/delve/pkg/config
github.com/go-delve/delve/vendor/go.starlark.net/syntax
github.com/go-delve/delve/vendor/github.com/mattn/go-colorable
github.com/go-delve/delve/pkg/dwarf/loclist
github.com/go-delve/delve/pkg/gobuild
github.com/go-delve/delve/pkg/dwarf/reader
github.com/go-delve/delve/pkg/proc/internal/ebpf
github.com/go-delve/delve/service/internal/sameuser
github.com/go-delve/delve/vendor/github.com/google/go-dap
github.com/go-delve/delve/vendor/github.com/inconshreveable/mousetrap
github.com/go-delve/delve/vendor/github.com/spf13/pflag
github.com/go-delve/delve/vendor/github.com/russross/blackfriday/v2
github.com/go-delve/delve/pkg/proc
github.com/go-delve/delve/vendor/go.starlark.net/resolve
github.com/go-delve/delve/vendor/go.starlark.net/internal/compile
github.com/go-delve/delve/vendor/github.com/cpuguy83/go-md2man/v2/md2man
github.com/go-delve/delve/vendor/github.com/spf13/cobra
github.com/go-delve/delve/vendor/go.starlark.net/starlark
github.com/go-delve/delve/vendor/github.com/spf13/cobra/doc
github.com/go-delve/delve/service/api
github.com/go-delve/delve/pkg/proc/amd64util
github.com/go-delve/delve/pkg/proc/winutil
github.com/go-delve/delve/pkg/proc/core/minidump
github.com/go-delve/delve/pkg/proc/native
github.com/go-delve/delve/pkg/proc/linutil
github.com/go-delve/delve/pkg/locspec
# github.com/go-delve/delve/pkg/proc/native
C:\Users\kbuilder\Go\src\github.com\go-delve\delve\pkg\proc\native\proc.go:49:5: cannot use &nativeProcess literal (type *nativeProcess) as type proc.ProcessInternal in assignment:
	*nativeProcess does not implement proc.ProcessInternal (missing GetBufferedTracepoints method)
C:\Users\kbuilder\Go\src\github.com\go-delve\delve\pkg\proc\native\proc_windows.go:494:59: undefined: proc.UProbeArgMap
github.com/go-delve/delve/pkg/proc/core
github.com/go-delve/delve/pkg/proc/gdbserial
```

--- a/pkg/proc/breakpoints.go
+++ b/pkg/proc/breakpoints.go
@@ -419,7 +419,7 @@ func (t *Target) SetEBPFTracepoint(fnName string) error {
 		if err != nil {
 			return err
 		}
-		offset, pieces, _, err := t.BinInfo().Location(entry, dwarf.AttrLocation, fn.Entry, op.DwarfRegisters{})
+		offset, pieces, _, err := t.BinInfo().Location(entry, dwarf.AttrLocation, fn.Entry, op.DwarfRegisters{}, nil)
 		if err != nil {
 			return err
 		}

--- a/pkg/proc/native/proc_windows.go
+++ b/pkg/proc/native/proc_windows.go
@@ -10,6 +10,7 @@ import (
 	sys "golang.org/x/sys/windows"
 
 	"github.com/go-delve/delve/pkg/proc"
+	"github.com/go-delve/delve/pkg/proc/internal/ebpf"
 )
 
 // osProcessDetails holds Windows specific information.
@@ -488,10 +489,15 @@ func (dbp *nativeProcess) EntryPoint() (uint64, error) {
 }
 
 func (dbp *nativeProcess) SupportsBPF() bool {
-	return true
+	return false
 }
 
-func (dbp *nativeProcess) SetUProbe(fnName string, args []proc.UProbeArgMap) {
+func (dbp *nativeProcess) SetUProbe(fnName string, args []ebpf.UProbeArgMap) error {
+	return nil
+}
+
+func (dbp *nativeProcess) GetBufferedTracepoints() []ebpf.RawUProbeParams {
+	return nil
 }
 
 func killProcess(pid int) error {

